sycl/test/matrix/runtime-query.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %clangxx -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -fsycl -o runtime-query %s
+// XFAIL: *
+
+#include <iostream>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental;
+using namespace sycl::ext::oneapi::experimental::matrix;
+
+template <matrix_type Ta, matrix_type Tb, matrix_type Tc, matrix_type Td>
+void matrix_runtime_query(queue q) {
+
+  std::vector<combination> combinations =
+      q.get_device().get_info<sycl::info::device::matrix_combinations>();
+
+  std::cout << "The matrix hardware implementation in this device provides "
+               "this number of combinations: "
+            << combinations.size() << std::endl;
+
+  bool max_sizes;
+  if (combinations[0].maxsize == 0)
```

Comment:
What is `maxsize`? There is no `maxsize` in `combination` struct. I will fix in the upcoming patch and enable (and re-write) the test for PVC, as this is a platform where we definitely test in this particular CI.